### ISSUE TITLE
Eyeball Segmentation w/Zarr, Paintera, Gunpowder

### ISSUE THREAD

**Issue by jmdelahanty** [Tyelab/bruker_pipeline#6]
Although tools like DLC/AlphaTracker could be used to label the point's around the eye/label eyelids, the nature of the problem lends itself better to perform segmentation of the eye with a binary mask through a neural network (I think...). This way we can not only get whether the animal is blinking/positions of their eyelids but also get data that can show how much surface area of the eye is exposed (approximately). There's a few things that must happen for a successful implementation:

- [x] Convert .avi to .zarr
- [x] Convert .zarr to .n5 temporarily
- [x] Create ground truth masks in Paintera
- [ ] Convert .n5 back to .zarr 
- [ ] Create `Gunpowder` pipeline
- [ ] Train network on cluster
- [ ] Check performance with Tensorboard (probably)
- [ ] Predict on other videos and see performance

Once these things are complete, we can go about incorporating an ML pipeline/update training cycle for the data and go from there...

**Comment by jmdelahanty** [comment]
There's a Python and MATLAB implementation that we could use for this kind of segmentation right away that's worth checking out. [See link here](https://github.com/fabiocarrara/meye)

**Comment by jmdelahanty** [comment]
Here's an example eye ball that was made a while back:
![eyball](https://user-images.githubusercontent.com/28411193/157586344-4c45729f-d7a4-4ecf-ab41-25e06d48434a.png)

**Comment by jmdelahanty** [comment]
Closing this since there's not a use case for it.
